packages/lib-classifier/src/store/FieldGuideStore.spec.js

```diff
@@ -35,62 +36,53 @@ const fieldGuideWithoutIcon = FieldGuideFactory.build({
 
 const project = ProjectFactory.build()
 
-describe('Model > FieldGuideStore', function () {
-  function fetchFieldGuide (rootStore) {
-    sinon.stub(rootStore.fieldGuide, 'fetchFieldGuide')
-    rootStore.projects.setResource(project)
-    return rootStore.projects.setActive(project.id)
-      .then(() => {
-        rootStore.fieldGuide.fetchFieldGuide.restore()
-        return rootStore.fieldGuide.fetchFieldGuide()
-      })
-  }
+function setupStores(clientStub) {
```

Comment:
This needs a comment to explain why the mocks for this store are set up differently from the other stores.